.circleci/config.yml

```diff
@@ -1,79 +1,105 @@
-jobs:
-  build:
-    environment:
-      COMMERCE_SDK_INPUT_DIR: ../test/raml/valid/
-    working_directory: ~/build_only
-    docker:
-      - image: circleci/node:10.16.3
-      - image: redis:5-alpine
+# DEFAULTS
+defaults: &defaults
+  working_directory: ~/build_only
+  environment:
+    COMMERCE_SDK_INPUT_DIR: ../test/raml/valid/
 
+# COMMANDS
+commands:
+  build:
     steps:
       - checkout
       - node/with-cache:
           steps:
             - run: npm ci
       - run: npm run build
+  test:
+    steps:
       - run: npm run test
       - run: npm run test:integration
       - store_test_results:
           path: reports
 
-  build-12:
-    environment:
-      COMMERCE_SDK_INPUT_DIR: ../test/raml/valid/
-    working_directory: ~/build_only
+# EXECUTORS
+executors:
+  node-10:
+    <<: *defaults
+    docker:
+      - image: circleci/node:10.16.3
+      - image: redis:5-alpine
+  node-12:
+    <<: *defaults
     docker:
       - image: circleci/node:12.16.1
       - image: redis:5-alpine
+  node-14:
+    <<: *defaults
+    docker:
+      - image: circleci/node:14
```

Comment:
We should be consistent about what version we use and should specify the full version here.